metadata-sheets-schema/radiomics-metadatasheet-schema.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "http://json-schema.org/draft-01/schema#",
+    "$id": "https://raw.githubusercontent.com/qbicsoftware/",
+    "title": "Radiomics metadata sheet",
+    "description": "A Metadata object representation for mapping the radiomics metadata sheet to OpenBIS metadata fields.",
+    "definitions": {
+        "Filenames": {
+            "type": "column",
+            "properties": {
+                "label": {
+                    "type": "string"
+                },
+                "value": {
+                    "type": "string"
+                }
+            },
+            "required": ["label","value"],
+            "OpenBIS_intance": "Q_BMI_IMAGING_DATA",
+            "OpenBIS_property": "Q_FILENAME"
+        }
```

Comment:
I am not sure if you want a definition here. If you look into https://github.com/qbicsoftware/metastats-object-model/blob/master/model.schema.json I used the definition to define how a condition looks like (they are more complex objects) and used the definition later in the schema. 
With  "type": "object", "properties": { ... } I list all the fields that are part of the schema like "Filenames" in your case. I think you mixed up the definitions part with the actual definition of the schema. You can look check this  https://json-schema.org/understanding-json-schema/ to learn more about how to create a schema. We can also meet to talk about it if you want.

Reply:
oh thanks @jenniferboedker for your feedback. You're totally right, I oversaw the first entry was different. I fix this and ask you again :)